docs/Radarr/Radarr-collection-of-custom-formats.md

```diff
@@ -1461,6 +1462,24 @@ We've made 3 guides related to this.
 
 ---
 
+### x266
+
+??? question "x266 - [Click to show/hide]"
+
+    Versatile Video Coding (VVC), also known as H.266, ISO/IEC 23090-3, and MPEG-I Part 3, is a video compression standard finalized on 6 July 2020. [Wikipedia](https://en.wikipedia.org/wiki/Versatile_Video_Coding){:target="_blank" rel="noopener noreferrer"}.
+
+    This applies to all x266/VVC releases that are not Remux.
```

Comment:
are there even remuxes using `x266`?

Reply:
I don't believe so - my assumption was that at the point x/h266 remuxes do exist then we wouldn't want to block them. The CF itself excludes remuxes. Happy to reconsider this though.

Reply:
hmm good point, probably no harm in keeping it in